content/all_datasets.md

```diff
@@ -60,6 +60,6 @@ before-content: gh_buttons.html
 ### Legend
 
 ¹ Labeling:
-- 🟩: Provides clear labels on at least a portion of the contained data
-- 🟨: Provides some form of ground truth that allows for manual labeling
-- 🟥: Does not provide labeled data or ground truth
\ No newline at end of file
+- 🟩: Direct; provides explicit labels on at least a portion of the contained data
+- 🟨: Indirect; provides some form of ground truth that allows for manual labeling
```

Comment:
This phrasing might be confusing since indirect labels may also be used for _automatic_ labeling.

Maybe: "Indirect; provides some form of ground truth that allows for manual or automatic labeling (e.g., periods of attack)"

Reply:
thx, changed